src/domain/model/idAuthentication/models.ts

```diff
@@ -0,0 +1,14 @@
+export class AuthenticationSessionToken {
+  constructor(readonly value: string) {}
+}
+
+export interface AuthenticationSession {
+  token: AuthenticationSessionToken;
+  redirectUrl: string;
+}
+
+export interface Authentication {
+  code: string;
+  firstName: string;
+  lastName: string;
```

Comment:
Bit of an odd interface?  How is it used?

Reply:
Is this the response we get from the 3rd party?  Maybe `AuthenticationResponse` or something?

Reply:
It is and good shout, it is odd